extensions/odoo_theme/static/scss/_sphinx_basic.scss

```
@@ -598,21 +598,19 @@ ol.upperroman {
 //     margin-bottom: 0px;
 // }
 
-ol.simple ol p,
-ol.simple ul p,
-ul.simple ol p,
-ul.simple ul p {
-    margin-top: 0;
-}
-
-ol.simple > li:not(:first-child) > p,
-ul.simple > li:not(:first-child) > p {
+ol.simple > li > p,
+ul.simple > li > p,
+ol.simple > li > ul > li > p,
+ul.simple > li > ul > li > p,
+ol.simple > li > ol > li > p,
+ul.simple > li > ol > li > p {
     margin-top: 0;
+    margin-bottom: 0;
 }
 
-ol.simple p,
-ul.simple p {
-    margin-bottom: 0;
+ul > li > ul.simple > li:last-child > p,
+ol > li > ul.simple > li:last-child > p {
```

As I said before, you can cover this case with a simpler selector :
```suggestion
li > .simple {
```
By doing this, it's easier to maintains the `.simple` spacing. 
You don't need to specify the paragraph margin to solve this issue.